docs_src/index.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-As Computer Vision becomes more and more mainstream, especially for industrial & retail use cases, development and deployment of these solutions becomes more challenging. Vision workloads are large and complex and need to go through many stages as shown in the figure below.
+As Computer Vision becomes more and more mainstream, especially for industrial & retail use cases, development and deployment of these solutions becomes more challenging. Vision workloads are large and complex and need to go through many stages. For instance, in the pipeline below, the video data is ingested, pre-processed before each inferencing step, inferenced using two models - YOLOv5 and EfficientNet, and post processed to generate metadata and show the bounding boxes for each frame.
 
```

Comment:
I feel, we should clarify here next line that the solution supports different other pipelines for object detection, classification & segmentation along with different other models. Just to be clear that solution is not restricted to just YOLOv5 & efficientNet. Earlier architecture diagram had generic AI model.

Reply:
I know @brian-intel mentioned there was some confusion with the earlier image because it was not necessarily representative of the solution. maybe we need to also look to see if there is a better generic architecture diagram we can implement as our solution grows.

Reply:
One idea is to have a page dedicated to the 3 main pipelines (yolov5, yolov5+efficientnet, yolov5+efficientnet+barcode+text detection+text recognition) and modify this to a generic diagram. Can change yolov5 to inference 1 and efficientnet to inference 2.